extension/src/background/helpers/dataStorage.ts

```diff
@@ -13,37 +13,60 @@ interface SetItemParams {
   [key: string]: any;
 }
 
-export const browserStorage = browser.storage?.local;
+// https://github.com/mozilla/webextension-polyfill/issues/424
+interface BrowserStorage extends browser.Storage.Static {
+  session: browser.Storage.LocalStorageArea;
+}
+
+const storage = browser.storage as BrowserStorage;
+
+// browser storage uses local storage which stores values on disk and persists data across sessions
+// session storage uses session storage which stores data in memory and clears data after every "session"
+// only use session storage for secrets or sensitive values
+export const browserStorage = storage?.local;
```

Comment:
I think now that we're starting to deal with 2 types of storage (1 long lived, 1 ephemeral) in many parts of the code, we should start using consistent naming to make it clear when you're saving something to the long lived store vs the short lived store.

why don't we call this var `localStorage`. And then, going forward in the code, every time we invoke `dataStorage` with `localStorage`, we call that `const localStore = dataStorage(localStorage)`. Similary, every time we're dealing with the short lived (either redux or storage.session), we call it `sessionStore`

Reply:
makes sense, will update the var names.